src/app/pages/transaction-request/components/submit-action.tsx

```diff
@@ -20,39 +19,37 @@ function BaseConfirmButton(props: ButtonProps): JSX.Element {
   );
 }
 
-function SubmitActionSuspense(): JSX.Element {
+interface SubmitActionProps {
+  hasFees: boolean;
```

Comment:
what does this mean exactly?

Reply:
The confirm button should become disabled when there is no fee selected or entered into the custom input. I will make this more clear and fix it.